src/ngraph/runtime/cpu/pass/cpu_assignment.cpp

```diff
@@ -125,6 +126,39 @@ namespace ngraph
                     }
                 }
 
+                template <>
+                void CPUAssignment::ASSIGN_DECL(ngraph::op::QuantizedConcat)
+                {
+                    auto quantized_concat = static_cast<op::QuantizedConcat*>(node);
+
+                    if ((node->get_input_element_type(0) == element::i8 ||
+                         node->get_input_element_type(0) == element::u8) &&
+                        ((node->get_input_shape(0)).size() == 4 ||
+                         (node->get_input_shape(0)).size() == 2))
```

Comment:
avoid code duplication. move this check to a can_use_mkldnn_concat() routine in mkldnn_utils.cpp.